src/python/WMComponent/DBS3Buffer/MySQL/Create.py

```diff
@@ -49,8 +49,9 @@ def __init__(self, logger=None, dbi=None, params=None):
                  subscribed             INTEGER      DEFAULT 0,
                  phedex_group           VARCHAR(100),
                  delete_blocks          INTEGER,
+                 dataset_lifetime           INTEGER      DEFAULT 0,
                  PRIMARY KEY (id),
-                 CONSTRAINT uq_dbs_dat_sub UNIQUE (dataset_id, site, custodial, auto_approve, move, priority))"""
+                 CONSTRAINT uq_dbs_dat_sub UNIQUE (dataset_id, site, custodial, auto_approve, move, priority, dataset_lifetime))"""
```

Comment:
Do we need to make dataset_lifetime part of the table unique constraint? I'm inclined to say no, but I might be missing something.